app/views/request_mailer/overdue_alert.text.erb

```diff
@@ -1,9 +1,16 @@
 <%= raw @info_request.public_body.name %> <%= _('have delayed.') %>
 
+<% if @info_request.public_body.not_subject_to_law? %>
+<%= _('They have not replied to your {{law_used_short}} request {{title}} ' \
+      'promptly',
+      :law_used_short => @info_request.law_used_human(:short).html_safe,
+      :title => @info_request.title.html_safe) %>.
+<% else %>
```

Comment:
I think this is adding complexity for no benefit. The `else` says "normally required by law", which would cover the `not_subject_to_law?` branch.

We can either:

1. Remove the conditional and keep the original wording, as that covers both cases
2. Be more forceful in the `else` branch and remove `"normally"`.

I'd vote for option 1.

Reply:
_selects option 1_